<?xml version="1.0" encoding="UTF-8" ?>

<messages cleanup="false" version="1000">
	<Message id="BPMPRO0001E" group="bpmpro" key="cannotconnected" prefix="1">
		<Display method="MSGBOX">
			<Option>MSG_BTNOK</Option>
		</Display>
		<MsgText>Endpoint could not connect to the server.</MsgText>
		<Explanation></Explanation>
		<AdminResponse></AdminResponse>
		<OperatorResponse></OperatorResponse>
	</Message>

	<Message id="BPMPRO0002E" group="bpmpro" key="autherror" prefix="1">
		<Display method="MSGBOX">
			<Option>MSG_BTNOK</Option>
		</Display>
		<MsgText>Endpoint could not authenticate the user.</MsgText>
		<Explanation></Explanation>
		<AdminResponse></AdminResponse>
		<OperatorResponse></OperatorResponse>
	</Message>

	<Message id="BPMPRO0003E" group="bpmpro" key="cannotcreatedir" prefix="1">
		<Display method="MSGBOX">
			<Option>MSG_BTNOK</Option>
		</Display>
		<MsgText>Endpoint could not create directories.</MsgText>
		<Explanation></Explanation>
		<AdminResponse></AdminResponse>
		<OperatorResponse></OperatorResponse>
	</Message>

	<Message id="BPMPRO0004E" group="bpmpro" key="ftpconnecterror" prefix="1">
		<Display method="MSGBOX">
			<Option>MSG_BTNOK</Option>
		</Display>
		<MsgText>Endpoint has encountered some network errors.</MsgText>
		<Explanation></Explanation>
		<AdminResponse></AdminResponse>
		<OperatorResponse></OperatorResponse>
	</Message>

	<Message id="BPMPRO0005E" group="bpmpro" key="ioexception" prefix="1">
		<Display method="MSGBOX">
			<Option>MSG_BTNOK</Option>
		</Display>
		<MsgText>File upload has failed.</MsgText>
		<Explanation></Explanation>
		<AdminResponse></AdminResponse>
		<OperatorResponse></OperatorResponse>
	</Message>

	<Message id="BPMPRO0006E" group="bpmpro" key="ftpunknownhost" prefix="1">
		<Display method="MSGBOX">
			<Option>MSG_BTNOK</Option>
		</Display>
		<MsgText>The specified hostname could not be recognized.</MsgText>
		<Explanation></Explanation>
		<AdminResponse></AdminResponse>
		<OperatorResponse></OperatorResponse>
	</Message>
</messages>
